examples/powertools-examples-idempotency/src/main/java/helloworld/App.java

```diff
@@ -65,6 +69,7 @@ public APIGatewayProxyResponseEvent handleRequest(final APIGatewayProxyRequestEv
         APIGatewayProxyResponseEvent response = new APIGatewayProxyResponseEvent()
                 .withHeaders(headers);
         try {
+            // Read the 'address' field from the JSON post body
```

Comment:
thinking about this example, I'm not sure this is the best example for idempotency, something with SQS would probably be wiser.

Reply:
I agree. But - I just want to touch documentation here - if we want to redo the examples to be structurally better i'd rather do that in a subsequent ticket. Otherwise i'm going to end up rewriting all the examples.

Reply:
yes, agreed